src/index.html

```diff
@@ -3,11 +3,46 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="card-product" data-qa="card">
+      <div class="card-product__image-container">
+        <div class="card-product__image"></div>
+      </div>
+      <h1 class="card-product__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card-product__code">
+        Product code: 195434
+      </p>
+      <div class="stars">
+        <div class="stars__rating-box">
+          <div class="stars__star stars__star--star-1"></div>
```

Comment:
remove this number modificator for stars _star element, and you can check implementation from stars task
```suggestion
          <div class="stars__star ></div>
```

Reply:
done